### PR TITLE
Lockup balance

### PR DESCRIPTION
**Pull request by vgrichina**
Fixes https://github.com/near/near-wallet/issues/670

### CODE REVIEW

File: src/utils/wallet.js [hunk 1]
```diff
@@ -379,11 +379,18 @@ class Wallet {
...
-        return await this.getAccount(userAccountId).getAccountBalance()
+        accountId = accountId || this.accountId
+
+        const account = this.getAccount(accountId)
+        const balance = await account.getAccountBalance()
+
+        // TODO: Should lockup contract balance be retrieved separately?
+        // TODO: Unhardcode lockup account name
+        const lockupAccountId = 'lockup-test.vg.betanet'
+        // TODO: Makes sense for a lockup contract to return whole state as JSON instead of method per property
```
Comment (bowenwang1996) [inline src/utils/wallet.js:390]: Do you suggest to modify the contract to add such a method? Right now rpc can only give state serialized in borsh

Comment (vgrichina) [reply src/utils/wallet.js:390]: @bowenwang1996 yeah, I suggest to modify the contract, otherwise it’s pretty inefficient to query the data

**Comment by vgrichina** [comment]
> Locked = total lockup balance - unvested balance - liquid owners balance
> Unvested = unvested balance
> Unlocked (can be withdrawn) = liquid owners balance

**Comment by vgrichina** [comment]
Check https://near-wallet-pr-735.onrender.com/profile/test-create-1.testnet to test

I can also create lockup for arbitrary accounts under `.lockup.vg`

**Comment by Patrick1904** [comment]
> Check https://near-wallet-pr-735.onrender.com/profile/test-create-1.testnet to test
> 
> I can also create lockup for arbitrary accounts under `.lockup.vg`

@vgrichina seems I'm unable to create an account (needed to view the profile).

**Comment by vgrichina** [comment]
> seems I'm unable to create an account

@Patrick1904 do you mean that you cannot create it at all or that it doesn't get lockup? (lockup account needs to be created manually outside of wallet)

**Comment by Patrick1904** [comment]
> > seems I'm unable to create an account
> 
> @Patrick1904 do you mean that you cannot create it at all or that it doesn't get lockup? (lockup account needs to be created manually outside of wallet)

I wasn't able to create an account at all but it worked now. Maybe it was a testnet issue. Looks good to me!

**Pull request merged**